terria/africa.json

```diff
@@ -243,6 +243,31 @@
                ]
             }
          ]
+      },
+      {
+         "name": "Ancillary Data",
+         "type": "group",
+         "preserveOrder": true,
+         "items": [
+            {
+               "name": "Digital Elevation Models",
+               "type": "group",
+               "preserveOrder": true,
+               "items": [
+                  {
+                     "name": "SRTM",
+                     "type": "wms",
+                     "layers": "srtm",
+                     "url": "https://ows.digitalearth.africa/",
+                     "linkedWcsUrl": "https://ows.digitalearth.africa/",
+                     "linkedWcsCoverage": "srtm",
+                     "ignoreUnknownTileErrors": true,
+                     "opacity": 1,
+                     "featureTimesProperty": "data_available_for_dates"
```

Comment:
please remove this line as it doesn't have multiple timesteps

Reply:
Done